tom_tns/forms.py

```diff
@@ -31,6 +31,7 @@ def is_set(self, field):
 class TNSReportForm(BaseReportForm):
     submitter = forms.CharField(required=True, widget=forms.HiddenInput())
     object_name = forms.CharField(required=True, widget=forms.HiddenInput())
+    telescope = forms.CharField(required=False, widget=forms.HiddenInput())
```

Comment:
I'm confused by this field. 
Each data point could be from a different telescope, so while we can guess which data is the discovery data, that doesn't necessarily mean that we know the telescope. Realistically, this should just be the bit before the dash in the TNS instrument field.

Reply:
For new discoveries or classifications with TNS you are generally only submitting a single datapoint (although its possible to do more than one, but not in the tom_tns form). So that one reduced datum comes from a single telescope. The TNS submission does not require telescope to be set, so it is a hidden field here that can be included when submitting through hermes, and it will show up in the hermes message but not in the TNS report.

Reply:
My point is that I can change the reduced Datum used as the discovery datum. If I do this in the form, it won't be reflected in the Hermes message. That's why it might be better to take the telescope from the TNS instrument choice field since that can be modified using the form.